src/IR/Types/Inference.hs

```diff
@@ -70,7 +70,13 @@ withBindings bs = U.local
 -- | Look up the 'U.Scheme' a data identifier in the variable environment.
 lookupBinding :: Identifiable i => i -> Infer U.Scheme
 lookupBinding x =
-  U.asks (M.lookup (fromId x) . varEnv) >>= maybe (U.throwError unbound) return
+  if fromId x == DConId "(,)"
+  then
+    do 
+    U.asks (M.lookup (fromId (DConId "Pair")) . varEnv) >>= maybe (U.throwError unbound) return
```

Comment:
Here is an example of where you can  replace "(,)" and "Pair" with identifiers defined in Identifiers.hs. We can talk more about this at our meeting.